index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <title>Set up and Mobile Version Skeletion</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" />
+
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+  <header>
+    <div class="nav-bar">
+      <a href="#">
+        My Logo
+      </a>
+      <i class="fas fa-bars"></i>
+    </div>
+  </header>
+  <main>
+    <section>
+      <div>
+        <div id="background-image"></div>
+        <div class="content">
+          <h1>I’m Mariane <br />
+            Glad to see <br /> you!</h1>
+          <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```

Comment:
- [ ] Kindly arrange the contents of this section using flex-box. It'll make implementations much easier when you decide to implement the desktop version :)